.gitignore

```diff
@@ -54,3 +54,5 @@ _checkouts
 
 .DS_Store
 zotonic.tmproj
+
+docker-data
```

Comment:
This directory isn't really needed.

Reply:
We have to leave the data from Docker somewhere - it seemed a good idea to me to provide a directory where all data is stored - so that you can also easily tune the configuration and security keys.

Reply:
It's good practice to mount the config directly in the expected place (`/etc/zotonic` in the case of Zotonic).

Reply:
You mean `/etc/zotonic` on the host system?

I would prefer to keep that data separate - the /etc/zotonic on the local system is for Zotonic running on the local system, not in some container.

Reply:
Same for the pg data.  It belongs to the container, not to the host system.

Reply:
No, `/etc/zotonic` in the container.

Reply:
It is in `/etc/zotonic` on the container, isn't it?

Reply:
At least that is where Zotonic picks it up.

That `/etc/zotonic` in the container is mapped to the local `docker-data/etc-zotonic`